training/sklearn/structured/hp_tuning/README.md

```diff
@@ -0,0 +1,187 @@
+# Training with scikit-learn - Hyperparameter Tuning
```

Comment:
Can we add more details as to which parameters we tune and the results of each run ?

Reply:
Added